Manual/ModuleSystem.lean

```diff
@@ -141,6 +143,13 @@ local elab "my_elab" : command => do
   ...
 ```
 
+For convenience, `meta` also implies `partial`.
+This can be overriden by giving an explicit `termination_by` metric (such as one suggested by `termination_by?`), which may be necessary when the type of the definition is not known to be `Nonempty`.
+
+As a guideline, it is usually preferable to apply `meta` as late as possible.
+Thus helper definitions not directly registered as metaprograms do not need to be marked if they are located in a separate module.
+Only the final module(s) actually registering metaprograms then should use `public meta import` to import those helpers and define its metaprograms using built-in syntax like `elab`, using `meta def`, or using `meta section`.
```

Comment:
```suggestion
Only the final module that actually registers a metaprogram needs the helpers as `meta` definitions.
It should use `public meta import` to import those helpers and then define its metaprograms using built-in syntax like `elab`, using `meta def`, or using `meta section`.
```
I think two sentences makes this easier to parse.

Can we motivate this? I see two reasons: that it enables re-use and independent non-meta testing of these helpers, and that things are generally just easier when not in `meta`-mode. Is there a third reason?